pilot/pkg/xds/ads.go

```diff
@@ -33,6 +33,7 @@ import (
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pilot/pkg/networking/util"
 	v3 "istio.io/istio/pilot/pkg/xds/v3"
+	"istio.io/istio/pkg/istio-agent/health"
```

Comment:
It is bad, we should never let pilot/istiod import from agent. 

Move the healthcheck type to pilot/pkg/xds/v3/model.go

Reply:
Should I import the TypeUrl in Istio-agent then?